<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Customer</class>
 <widget class="QWidget" name="Customer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>505</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#frame{
border-image: url(:/lantianlou.jpg);}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>505</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文行楷</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>小青梅汉服</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>741</width>
      <height>421</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文中宋</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>款式：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文中宋</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>品牌：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Cus_queryTypeComboBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>160</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="Cus_queryBrandComboBox">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>160</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Cus_queryPushButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>220</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>220</width>
        <height>60</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="imageLabel">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>60</y>
        <width>300</width>
        <height>360</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Cus_sellOkBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>250</y>
        <width>220</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>购买</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>350</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文中宋</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>数量：</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="Cus_NumSpinBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>340</y>
        <width>160</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
